src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/BundlesRespondentController.java

```diff
@@ -228,6 +228,70 @@ public ResponseEntity<CCDCallbackResponse> submitted(
                 .build());
     }
 
+    @PostMapping(value = "/removeHearingBundle", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Remove a hearing bundle")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Removed successfully",
+            content = {
+                @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = CCDCallbackResponse.class))
+            }),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> removeHearingBundle(
+            @RequestBody CCDRequest ccdRequest,
+            @RequestHeader("Authorization") String userToken) {
+
+        throwIfBundlesFlagDisabled();
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            log.error(INVALID_TOKEN, userToken);
+            return ResponseEntity.status(FORBIDDEN.value()).build();
+        }
```

Comment:
```suggestion
```
no longer need these on ET as we have a request interceptor that will do this for us